gui-lib/mred/private/wx/gtk/group-panel.rkt

```diff
@@ -6,6 +6,7 @@
          "window.rkt"
          "client-window.rkt"
          "panel.rkt"
+         "frame.rkt"
```

Comment:
Doesn't seem like a big deal, but is this import needed?

Reply:
It's needed because they were getting the functions from the other definition before.

Reply:
Oh, I see. Ok!